ush/config_defaults.sh

```diff
@@ -683,6 +683,8 @@ binary_diag=.true.
 
 # &HYBRID_ENSEMBLE
 l_both_fv3sar_gfs_ens=.false.
+weight_ens_gfs=1.0
+weight_ens_fv3sar=1.0
```

Comment:
The default value of weight_ens_fv3sar should be 0.0?

Reply:
If the default value is 0.0, FV3LAM ensemble is not used for ensemble B at all even in activating MixEn. This specification is different from the current MixEn. I think it should be 1.0 to keep backward compatibility for MixEn.

Reply:
Thanks for the explanation. Although it's a little bit confusing, I got what it does.